spec/views/catalog/show.html.erb_spec.rb

```diff
@@ -4,9 +4,13 @@
   let(:document) { SolrDocument.new :id => 'xyz', :format => 'a' }
   let(:title) { 'Long title that should be truncated at 50 characters' }
   let(:blacklight_config) { Blacklight::Configuration.new }
+  let(:query_params) { { controller: 'catalog', action: 'show' } }
+  let(:search_state) { Blacklight::SearchState.new(query_params, blacklight_config) }
   before(:each) do
     assign :document, document
     allow(view).to receive(:blacklight_config).and_return(blacklight_config)
+    allow(view).to receive(:search_state).and_return search_state
+    allow(view).to receive(:application_name).and_return 'Argo'
```

Comment:
when i added the `search_state` stubbing, but not this line, i got this error:
```
  1) catalog/show.html.erb assigns page title, truncating it
     Failure/Error:
       expect(view.instance_variable_get(:@page_title))
         .to eq 'Long title that should be truncated at 50 chara... - Argo'
     
       expected: "Long title that should be truncated at 50 chara... - Argo"
            got: "Long title that should be truncated at 50 chara... - Blacklight"
     
       (compared using ==)
     # ./spec/views/catalog/show.html.erb_spec.rb:22:in `block (2 levels) in <top (required)>'
```

again, slightly uncomfortable with this change due to lack of understanding, because i was reusing the same `blacklight_config` var as was already working with the test.  but, as this is a view spec, it seemed that the point was not to test the configuration done in the rest of the app, and it seemed defensible to stub the `application_name` result.